Server/src/routes/authenticationRouter.ts

```diff
@@ -12,23 +12,27 @@ let authenticationController = new AuthenticationController();
  */
 
 router.post('/signup', (request: Request, response: Response, next: NextFunction) => {
-    authenticationController.createSignUpRequest(request, response, next);
+  authenticationController.createSignUpRequest(request, response, next);
 });
 
 router.post('/login', (request: Request, response: Response, next: NextFunction) => {
-    authenticationController.createLoginRequest(request, response, next);
+  authenticationController.createLoginRequest(request, response, next);
 });
 
-router.post('/api/v1/resetpassword', JWTAuthenticator.verifyJWT, async (request: Request, response: Response, next: NextFunction) => {
-    authenticationController.createPasswordResetRequest(request, response, next);
+router.post('/forgotPassword', (request: Request, response: Response, next: NextFunction) => {
```

Comment:
This naming convention goes against what we have transitioned to. All routes now should be in this format: /api/v1/myRouteHere
Right now we are on v1, but as overhauls happen, this will change to v2, v3, etc.

Reply:
Perfect, done